src/dbxdeploy/repos/RepoManager.py

```diff
@@ -77,10 +93,13 @@ def __check_duplicates(self, repo_list: list):
         if len(duplicated_branches) > 0:
```

Comment:
__check_duplicates() should also check tags, not only branches

Reply:
Tag key not supperted in list-repo Repos Api. Will revert to branches only.